src/core/cs3iface.py

```diff
@@ -160,8 +166,8 @@ def getxattr(endpoint, filepath, userid, key):
         log.debug('msg="Invoked stat for getxattr" filepath="%s" elapsedTimems="%.1f"' % (filepath, (tend - tstart) * 1000))
         return xattrvalue
     except KeyError:
-        log.warning('msg="Empty value or key not found in getxattr" filepath="%s" key="%s" trace="%s" metadata="%s"' %
-                    (filepath, key, statInfo.status.trace, statInfo.info.arbitrary_metadata.metadata))
+        log.info('msg="Empty value or key not found in getxattr" filepath="%s" key="%s" trace="%s" metadata="%s"' %
```

Comment:
Why do you change the log level here?

Reply:
This is because looking for non-existing attributes routinely happens in the code, so it's not really a warning condition. Same as for stats that return ENOENT.